src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/SafeFireAndForgetExtensions.shared.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Threading.Tasks;
+
+// Inspired by https://github.com/brminnick/AsyncAwaitBestPractices
```

Comment:
Just a question. Don't it violates the CLA? I mean if we contribute something, we provide it FOC (And don't put our names anywhere cuz Microsoft owns it).

CC @jfversluis

Reply:
This is included because the code was copied from AsyncAwaitBestPractices which is under the MIT license and requires attribution. I don't want anyone thinking we stole code from another library without attribution 😃.

Reply:
Hm, I guess MIT allows us to do with code whatever we want even w/out mentioning original authors)